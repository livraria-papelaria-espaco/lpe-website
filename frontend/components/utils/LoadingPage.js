import { CircularProgress } from '@material-ui/core';
import { makeStyles } from '@material-ui/core/styles';
import React from 'react';

const useStyles = makeStyles({
  root: {
    display: 'flex',
    width: '100%',
    height: '70vh',
    alignItems: 'center',
    justifyContent: 'center',
  },
});

const LoadingPage = () => {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <CircularProgress />
    </div>
  );
};

export default LoadingPage;
