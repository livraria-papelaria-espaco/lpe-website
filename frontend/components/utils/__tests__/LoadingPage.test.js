import React from 'react';
import { shallow } from 'enzyme';
import LoadingPage from '../LoadingPage';

describe('LoadingPage Utility (components/utils/LoadingPage)', () => {
  it('Renders without crashing and matches snapshot', () => {
    const wrapper = shallow(<LoadingPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
