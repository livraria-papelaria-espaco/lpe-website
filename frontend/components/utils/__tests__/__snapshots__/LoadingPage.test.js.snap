// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingPage Utility (components/utils/LoadingPage) Renders without crashing and matches snapshot 1`] = `
<div
  className="makeStyles-root-1 makeStyles-root-2"
>
  <WithStyles(ForwardRef(CircularProgress)) />
</div>
`;
